src/Qowaiv.Diagnostics.Contracts/README.md

```diff
@@ -9,8 +9,7 @@ should add the dependency as follows:
   <PackageReference
     Include="Qowaiv.Diagnostics.Contracts"
     Version="*"
```

Comment:
Is there more to change to the readme now these changes are made?

Reply:
Should be fine. The changes I made to the `ProjectReferences` are only specific to local projects (otherwise the source generators don't fire). But this is fine if the project is imported from NuGet